pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -70,7 +74,8 @@ var ErrNonSupportedControllerType = errors.New("Disk is attached to non-supporte
 
 // VSphere is an implementation of cloud provider Interface for VSphere.
 type VSphere struct {
-	cfg *VSphereConfig
+	client govmomi.Client
```

Comment:
Do you want to import govmomi.Client at the top instead of using the govmomi package name here?